data/text.json

```diff
@@ -0,0 +1 @@
+["publiclab", "gsoc"]
```

Comment:
One more change, if possible -- could we put this in `/spec/javascripts/fixtures/` or perhaps better yet in `/examples/data/`?